pkg/minikube/node/start.go

```diff
@@ -156,9 +156,11 @@ func Start(starter Starter, apiServer bool) (*kubeconfig.Settings, error) {
 			return nil, errors.Wrap(err, "Updating node")
 		}
 
-		cpBs, err := cluster.Bootstrapper(starter.MachineAPI, viper.GetString(cmdcfg.Bootstrapper), *starter.Cfg, starter.Runner)
+		// Make sure to use the command runner for the control plane to generate the join token
+		c := mustload.Running(starter.Cfg.Name)
```

Comment:
`mustload` should only be called from `main` - as it may panic. Call `machine.CommandRunner` instead.

Reply:
done